client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SecurityDocumentationIT.java

```diff
@@ -68,15 +71,104 @@ public void onFailure(Exception e) {
+
+        {
+            //tag::enable-user-execute-listener
+            EnableUserRequest request = new EnableUserRequest("enable_user", RefreshPolicy.NONE);
+            ActionListener<EmptyResponse> listener = new ActionListener<EmptyResponse>() {
+                @Override
+                public void onResponse(EmptyResponse setUserEnabledResponse) {
+                    // <1>
+                }
+
+                @Override
+                public void onFailure(Exception e) {
+                    // <2>
+                }
+            };
+            //end::enable-user-execute-listener
+
+            // Replace the empty listener by a blocking listener in test
+            final CountDownLatch latch = new CountDownLatch(1);
+            listener = new LatchedActionListener<>(listener, latch);
+
+            // tag::enable-user-execute-async
+            client.security().enableUserAsync(request, RequestOptions.DEFAULT, listener); // <1>
+            // end::enable-user-execute-async
+
+            assertTrue(latch.await(30L, TimeUnit.SECONDS));
+        }
+    }
+
+    public void testDisableUser() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+        char[] password = new char[]{'p', 'a', 's', 's', 'w', 'o', 'r', 'd'};
+        PutUserRequest putUserRequest = new PutUserRequest("disable_user", password, Collections.singletonList("superuser"), null,
+            null, true, null, RefreshPolicy.IMMEDIATE);
+        PutUserResponse putUserResponse = client.security().putUser(putUserRequest, RequestOptions.DEFAULT);
+        assertTrue(putUserResponse.isCreated());
+        {
+            //tag::disable-user-execute
+            DisableUserRequest request = new DisableUserRequest("disable_user", RefreshPolicy.NONE);
+            EmptyResponse response = client.security().disableUser(request, RequestOptions.DEFAULT);
+            //end::disable-user-execute
+
+            assertNotNull(response);
+        }
+
+        {
+            //tag::disable-user-execute-listener
+            DisableUserRequest request = new DisableUserRequest("disable_user", RefreshPolicy.NONE);
+            ActionListener<EmptyResponse> listener = new ActionListener<EmptyResponse>() {
+                @Override
+                public void onResponse(EmptyResponse setUserEnabledResponse) {
+                    // <1>
+                }
+
+                @Override
+                public void onFailure(Exception e) {
+                    // <2>
+                }
+            };
+            //end::disable-user-execute-listener
+
+            // Replace the empty listener by a blocking listener in test
+            final CountDownLatch latch = new CountDownLatch(1);
+            listener = new LatchedActionListener<>(listener, latch);
+
+            // tag::disable-user-execute-async
+            client.security().disableUserAsync(request, RequestOptions.DEFAULT, listener); // <1>
+            // end::disable-user-execute-async
```

Comment:
Do you think it would be suitable to add an enable/disable test for a missing user?

Reply:
I think this should be covered in tests for the API itself and not in our documentation tests, so I will leave this as is now.